css-attribute-case-sensitivity/index.html

```diff
@@ -0,0 +1,36 @@
+---
+feature_name: Case-insensitive Attribute Selector Matching
+chrome_version: 49
+feature_id: 5610936115134464
+---
+
+<h3>Background</h3>
+<p>To allow an author to match a CSS attribute (ASCII) case-insensitively, an
```

Comment:
Isn't this a match against an HTML attribute, not a CSS attribute? Also, what's with the `(ASCII)` parenthetical aside? I was [under the impression](https://mathiasbynens.be/notes/html5-id-class) that HTML attributes could contain Unicode characters.